tests/mock_network/churn.rs

```diff
@@ -50,7 +286,7 @@ fn drop_random_nodes<R: Rng>(
         .collect();
     let mut drop_count: BTreeMap<_, _> = sections.keys().map(|pfx| (*pfx, 0)).collect();
     loop {
-        let i = gen_range(rng, 1, nodes.len());
+        let i = gen_range(rng, 0, nodes.len());
```

Comment:
removing the first node could be dangerous.
there were many failures related to it previously

Reply:
We should be able to remove even the first node, so if it causes issues it's a bug that should be addressed rather than avoided. Also the first node isn't that special anymore. The only difference from normal node is that it accepts bootstrap requests even when the section size is less than `elder_size` while other nodes don't. I'm not even sure why we even have that distinction...

Reply:
> I'm not even sure why we even have that distinction
I think that is just a hangover from the differentiating first node. We should not have that any more. Now any node should allow bostrap requests. It's muhc simpler now.

Reply:
Addressed by https://github.com/maidsafe/routing/pull/1985